### PR TITLE
Update to Audacity 2.2.0

### PR DESCRIPTION
**Pull request by jurf**
I don’t mind keeping care of this package so expect future changes, this PR is just because I want 2.2.0 out and I won’t have time to look into this for at least two weeks.

### CODE REVIEW

**Comment by jurf** [comment]
bot, build

**Comment by jurf** [comment]
bot, build

**Comment by Barthalion** [comment]
You need to remove `configure` for libid3tag and run autoreconf.

**Comment by alexlarsson** [comment]
That sounds like overkill, you just need to copy in a new config.guess:
```
                 {
                     "type": "shell",
                     "commands": [
                         "cp /usr/share/gnu-config/config.sub .",
                         "cp /usr/share/gnu-config/config.guess ."
                     ]
                 }
```

File: org.audacityteam.Audacity.json [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-    "runtime-version": "3.24",
+    "runtime-version": "3.26",
     "sdk": "org.gnome.Sdk",
     "command": "audacity",
     "finish-args": [
         "--share=ipc",
         "--socket=x11",
         "--socket=pulseaudio",
-        "--filesystem=host"
+        "--filesystem=home"
```
Comment (TingPing) [inline org.audacityteam.Audacity.json]: Honestly leaving host probably makes sense. I doubt Audacity will ever support portals or anything.

Comment (jurf) [reply org.audacityteam.Audacity.json]: I remember something failing to launch when giving access host (I think it was scribus) so I just subconsciously use home, but I’ll change it back.

**Comment by jurf** [comment]
Thanks for looking into it guys, hope it builds this time.

bot, build

**Comment by jurf** [comment]
bot, build

**Comment by jurf** [comment]
Yay, it built. Thanks.

**Pull request merged**